### PR TITLE
174 incorporate generateassertionssummary into dos dashboard

### LINKED ISSUE #174: Incorporate GenerateAssertionsSummary into DoS dashboard
For incorporation into UI.

### PR DESCRIPTION
**Pull request by vteague**
This makes a table that combines the generate assertions summary from the database (as returned in the DosDashboardRefreshResponse) with the status returned by http.

Also some test class cleanup.

### CODE REVIEW

**Review by cjcarlton — APPROVED** [review]
looks good! i'll continue work on the page in case 94

File: server/eclipse-project/src/main/java/us/freeandfair/corla/json/DoSDashboardRefreshResponse.java [hunk 1]
```diff
@@ -111,20 +112,25 @@ public class DoSDashboardRefreshResponse {
...
 
+  /**
+   * The generate assertions summaries, for IRV contests. Keyed by contest name (which is repeated
+   * in the GenerateAssertionsSummary).
+   */
+  private final List<GenerateAssertionsSummary> my_generate_assertions_summaries;
+
   /**
    * Constructs a new DosDashboardRefreshResponse.
    *
```
Comment (michelleblom) [inline server/eclipse-project/src/main/java/us/freeandfair/corla/json/DoSDashboardRefreshResponse.java:123]: It looks like the parameters in the comments don't really match those in the method.

File: server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertionsAPITests.java [hunk 2]
```diff
@@ -86,13 +85,13 @@ public class GenerateAssertionsAPITests extends TestClassWithAuth {
   private final GenerateAssertions endpoint = new GenerateAssertions();
 
   /**
-   * Mock response for tinyExample1 contest
+   * Mock response for tinyExample1 contest, which succeeded and is not worth retrying.
    */
   private final static GenerateAssertionsResponse tinyIRVResponse
       = new GenerateAssertionsResponse(tinyIRV, true, false);
```
Comment (michelleblom) [inline server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertionsAPITests.java:92]: Not clear what we mean by "boring" parameters!

**Review by michelleblom — APPROVED** [review]

File: server/eclipse-project/src/main/java/us/freeandfair/corla/json/DoSDashboardRefreshResponse.java (hunk 1 above)
Comment (vteague) [reply server/eclipse-project/src/main/java/us/freeandfair/corla/json/DoSDashboardRefreshResponse.java:123]: Now fixed.

File: server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertionsAPITests.java (hunk 2 above)
Comment (vteague) [reply server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertionsAPITests.java:92]: I was trying to convey the idea that this is not a test for funny edge cases of those parameters.

Comment (vteague) [reply server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/endpoint/GenerateAssertionsAPITests.java:92]: Have changed the wording slightly. In general it's a good question to think about how to succinctly convey "in this test the parameter I actually care about is..."
